demo/src/app/components/accordion/accordion.component.html

```diff
@@ -1,8 +1,8 @@
-<ngbd-component-docs [name]="accordionDocs.name"
+<ngbd-component-docs [name]="accordionDocs.selector"
```

Comment:
I don't think we want to recommend people to import concrete components by `fileName`.

Better to output scoped packages.